packages/devextreme/js/__internal/core/license/license_validation.test.ts

```diff
@@ -507,3 +511,27 @@ describe('internal license check', () => {
     expect(trialPanelSpy).not.toHaveBeenCalled();
   });
 });
+
+describe('assertedVersions integration', () => {
+  it('assertDevExtremeVersion config().assertedVersions', () => {
```

Comment:
The rest of the file uses `test`, not `it`, so I suggest sticking with `test` to maintain consistency.
Btw, I think that `it` is generally intended for BDD-style assertions and assumes writing test names in [BDD style](https://en.wikipedia.org/wiki/Behavior-driven_development#Behavioral_specifications)